operator/v1/types_ingress.go

```diff
@@ -211,6 +226,31 @@ type IngressControllerStatus struct {
 
 	// endpointPublishingStrategy is the actual strategy in use.
 	EndpointPublishingStrategy EndpointPublishingStrategy `json:"endpointPublishingStrategy"`
+
+	// conditions is a list of conditions and their status.
+	//
+	//   * LoadBalancerManaged
+	//   - True if the endpoint publishing strategy requires a service load balancer.
+	//   - False if the endpoint publishing strategy does not require a service load
+	//     balancer.
+	//
+	//   * LoadBalancerReady
+	//   - True if a load balancer is managed and if the service
+	//     .status.loadBalancer.ingress hostname has been assigned.
+	//   - False if a load balancer is managed and the service
+	//     .status.loadBalancer.ingress has not been assigned.
+	//
+	//   * DNSManaged
+	//   - True if the endpoint publishing strategy and platform supports DNS
+	//     integration and dns.config.openshift.io/cluster configures DNS zones.
+	//   - False if the endpoint publishing strategy or platform doesn't support
+	//     DNS management, the ingress controller domain is unset, or if
+	//     dns.config.openshift.io/cluster doesn't define any managed zones.
```

Comment:
How about the following?
```
	//   - True if the following conditions are met:
	//     * The endpoint publishing strategy and platform support DNS.
	//     * The ingress controller domain is set
	//     * dns.config.openshift.io/cluster configures DNS zones.
	//   - False if any of those conditions are unsatisfied.
```

Reply:
Good idea — fixed here and also applied the format to other conditions